server/selinux/server/pulp-celery.te

```diff
@@ -55,6 +55,8 @@ allow celery_t pulp_tmp_t:file manage_file_perms;
 files_tmp_filetrans(celery_t, pulp_tmp_t, file)
 
 allow celery_t pulp_var_run_t:file manage_file_perms;
+# celery searches the directory for any existing pid files when it starts
+allow celery_t pulp_var_run_t:dir manage_dir_perms;
```

Comment:
The more I think about this; I think reverting this to the more restrictive version would be better. Sorry for the back and forth.